src/hotspot/share/gc/g1/g1CollectedHeap.cpp

```diff
@@ -1889,9 +1889,7 @@ bool G1CollectedHeap::try_collect_concurrently(GCCause::Cause cause,
     } else if (!GCCause::is_user_requested_gc(cause)) {
       assert(cause == GCCause::_g1_humongous_allocation ||
              cause == GCCause::_g1_periodic_collection ||
-             cause == GCCause::_wb_breakpoint ||
-             cause == GCCause::_java_lang_system_gc ||
-             cause == GCCause::_dcmd_gc_run, "Unsupported cause %s", GCCause::to_string(cause));
+             cause == GCCause::_wb_breakpoint, "Unsupported cause %s", GCCause::to_string(cause));
```

Comment:
`_wb_breakpoint` is unexpected here too, since it's completely handled by an earlier clause in the if-ladder.
The first two are the entire residue that should be able to get here.

Reply:
fixed. Thanks for pointing this out.